src/fonts/font-face.css

```diff
@@ -0,0 +1,9 @@
+
+@font-face {
```

Comment:
I started by sourcing only one font at 500 thickness but we can bring in more as well in a future pr.